cassandra/src/test/scala/akka/stream/alpakka/cassandra/scaladsl/README.txt

```diff
@@ -0,0 +1,7 @@
+All the tests must be run with a local Cassandra running on default port 9042.
+
+When using docker, a temporary server can by started by running:
+
+```
+docker run --rm -p 9042:9042 cassandra:3
+```
```

Comment:
We have this information in the paradox docs and in scaladoc of the spec classes. I think we can remove these `txt` files.